frontend/src/reducers/parcelsReducer.spec.tsx

```diff
@@ -29,6 +29,7 @@ const defaultParcelDetail: IParcelDetail = {
     longitude: 1,
     latitude: 2,
     id: 3,
+    fiscals: [],
```

Comment:
Does it also needs evaluations?

Reply:
it already had evaluations. more importantly this auto-merge doesn't make sense given a previous change, will remove this line.